jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java

```diff
@@ -177,9 +177,19 @@ private List<Map<String, Object>> fetchEntitiyInfoFromS3(String datasource, Stri
         LOGGER.debug("s3Role:{}", s3Role);
         LOGGER.debug("bucketName:{}", bucketName);
         LOGGER.debug("sourceFolder:{}", sourceFolder);
+        LOGGER.debug("datasource:{}", datasource);
         try {
-            //override datapath with sourceFolder
-            dataPath = sourceFolder;
+            //override datapath with sourceFolder(i.e) for redhat s3.data will come from Cq-mapper
+            if (System.getenv("s3.data")!=null && !System.getenv("s3.data").isEmpty()) {
```

Comment:
I thought the data in params object get set as env variable? Hence no need for this as even for redhat scenario, since data is in s3.data field will come in the params just like other events from event bridge?

Reply:
@plyubich we had discussion internally with @arunpaladin and @akashbabu719 and decided to go with this. If required we can have call tommorow and sort this out